sky/setup_files/setup.py

```diff
@@ -56,7 +56,9 @@ def parse_footnote(readme: str) -> str:
 
 install_requires = [
     'wheel',
-    'Click',
+    # NOTE: ray 1.13.0 requires click<=8.0.4,>=7.0; our use of shell_complete
+    # requires >= 8.
+    'click>=8,<=8.0.4',
```

Comment:
Use `>=7.0<=8.0.4` instead, as we support it with the changes in this PR?

Reply:
I see! I've also updated the docs to reflect this.